app/models/application.rb

```diff
@@ -5,14 +5,22 @@ class Application < ActiveRecord::Base
   has_many :steps
 
   def self.statuses
-    %W(open waiting dead icebox offered)
+    %W(to-apply in-progress applied closed )
   end
 
-  def self.active
-    where.not(:status => 'dead').order(:company)
+  def self.to_apply
+    where(status: 'to-apply').order(:company)
   end
 
-  def self.dead
-    where(:status => 'dead').order(:company)
+  def self.in_progress
+    where(status: 'in-progress').order(:company)
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.